lib/Service/BinExt.php

```diff
@@ -76,7 +76,7 @@ public static function testExiftool(): string
         $version = trim($out);
         $target = self::EXIFTOOL_VER;
         if (!version_compare($version, $target, '=')) {
-            throw new \Exception("version does not match {$version} <==> {$target}");
+            throw new \Exception("exiftool version does not match: expected {$target} but found {$version}");
```

Comment:
the binary name is redundant, IIRC. The UI already shows it

Reply:
This message is also shown in occ which has no context around it.